node-problem-detector.yaml

```diff
@@ -26,6 +26,11 @@ spec:
         - name: log
           mountPath: /log
           readOnly: true
+        # Make sure node problem detector is in the same timezone
+        # with the host.
+        - name: localtime
+          mountPath: /etc/localtime
+          readOnly: true
```

Comment:
That's wired. When I initially decided to go with this approach, I remembered I've tried the symlink situation, it worked.

I'll try again then.